packages/path_provider/path_provider_linux/test/path_provider_linux_test.dart

```diff
@@ -16,6 +24,12 @@ void main() {
   test('getTemporaryPath', () async {
     final PathProviderPlatform plugin = PathProviderPlatform.instance;
     expect(await plugin.getTemporaryPath(), '/tmp');
```

Comment:
Unless I'm missing something, this will fail if someone runs it on a machine with TMPDIR set to something non-default. If you want to explicitly test null, you should be using an environment created by the test that doesn't contain TMPDIR, rather than assuming the host environment doesn't have it.

Reply:
We pass `Map<String, String>` as environment , which is initially empty. Therefore, there can be no conflict.